build issue -

**Description**

Compile error when building from source repo using "make"

**Steps to reproduce the issue:**

1. clone repo
2. install make, go, qemu
3. run make in repo folder

**Describe the results you received:**

linuxkit git:(master) make
tar cf - -C src/cmd/linuxkit . | docker run --rm --net=none --log-driver=none -i -e GOOS=darwin linuxkit/go-compile:b1446b2ba407225011f97ae1dba0f512ae7f9b84 --package github.com/linuxkit/linuxkit/src/cmd/linuxkit --ldflags "-X github.com/linuxkit/linuxkit/src/cmd/linuxkit/version.GitCommit=ffea2a280780535a85fab1b36eb256ec3e3ef6f2 -X github.com/linuxkit/linuxkit/src/cmd/linuxkit/version.Version="v0.8+"" -o bin/linuxkit > tmp_linuxkit_bin.tar
Unable to find image 'linuxkit/go-compile:b1446b2ba407225011f97ae1dba0f512ae7f9b84' locally
b1446b2ba407225011f97ae1dba0f512ae7f9b84: Pulling from linuxkit/go-compile
18f5a4610dcd: Pulling fs layer
1fcd32c88d57: Pulling fs layer
578f42885187: Pulling fs layer
578f42885187: Verifying Checksum
578f42885187: Download complete
1fcd32c88d57: Verifying Checksum
1fcd32c88d57: Download complete
18f5a4610dcd: Verifying Checksum
18f5a4610dcd: Download complete
18f5a4610dcd: Pull complete
1fcd32c88d57: Pull complete
578f42885187: Pull complete
Digest: sha256:4c8e0936d283d77ce5756e75cf4a7c78cd1dfc9908491ce4a5234899fc4d2b8c
Status: Downloaded newer image for linuxkit/go-compile:b1446b2ba407225011f97ae1dba0f512ae7f9b84
gofmt...
govet...
golint...
./pkglib/manifest.go:35:100: func parameter plainHttp should be plainHTTP
sh: func: unknown operand
make: *** [tmp_linuxkit_bin.tar] Error 2
make: *** Deleting file `tmp_linuxkit_bin.tar'

**Describe the results you expected:**

build success

**Additional information you deem important (e.g. issue happens only occasionally):**

Tried on macos and on Debian 10 with the same result

--- a/src/cmd/linuxkit/pkglib/manifest.go
+++ b/src/cmd/linuxkit/pkglib/manifest.go
@@ -32,7 +32,7 @@ import (
  push.go and util.go. It then was modified to remove any command-line dependencies.
 */
 
-func pushManifestList(auth dockertypes.AuthConfig, input types.YAMLInput, ignoreMissing, insecure, plainHttp bool, configDir string) (hash string, length int, err error) {
+func pushManifestList(auth dockertypes.AuthConfig, input types.YAMLInput, ignoreMissing, insecure, plainHTTP bool, configDir string) (hash string, length int, err error) {
 	// resolve the target image reference for the combined manifest list/index
 	targetRef, err := reference.ParseNormalizedNamed(input.Image)
 	if err != nil {
@@ -44,7 +44,7 @@ func pushManifestList(auth dockertypes.AuthConfig, input types.YAMLInput, ignore
 		configDirs = append(configDirs, filepath.Join(configDir, "config.json"))
 	}
 	resolver := newResolver(auth.Username, auth.Password, insecure,
-		plainHttp, configDirs...)
+		plainHTTP, configDirs...)
 
 	imageType := types.Docker
 	manifestList := types.ManifestList{

